api/openapi/api.yaml

```diff
@@ -631,7 +631,7 @@ paths:
       operationId: api.routes.get_project_manifests
       parameters:
         - in: query
-          name: input_token
```

Comment:
I am a little bit concerned of changing all the `input_token` to `access_token`. That means all the downstream applications using schematic APIs need to update their syntax.

Reply:
Yeah, we'd need to coordinate with those users like we are with @afwillia to update the endpoints, but I think this would result in a descriptive parameter name while also eliminating duplicate parameters.